rules/windows/lateral_movement_incoming_wmi.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2020/11/15"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies processes executed via Windows Management Instrumentation (WMI) on a remote host. This could be indicative of
+adversary lateral movement but may be noisy if administrators use WMI to remotely manage hosts.
```

Comment:
```suggestion
Identifies processes executed via Windows Management Instrumentation (WMI) on a remote host. This could be indicative of
adversary lateral movement, but could be noisy if administrators use WMI to remotely manage hosts.
```